docs/data/date-pickers/accessibility/accessibility.md

```diff
@@ -0,0 +1,84 @@
+|   <kbd class="key">Page Up</kbd> | Moves calendar to next month, keeping focus on the same day     |
+| <kbd class="key">Page Down</kbd> | Moves calendar to previous month, keeping focus on the same day |
+|      <kbd class="key">Home</kbd> | Moves focus to the first day of the week                        |
+|       <kbd class="key">End</kbd> | Moves focus to the last day of the week                         |
+
+### Date Picker
+
+The [Date Picker](/x/react-date-pickers/date-picker/) combines the functionalities of the Date Field and Date Calendar components.
+
+Depending on which component is in focus, the Picker will provide the corresponding keyboard support, either from [Date Field](/x/react-date-pickers/accessibility/#fields) or [Date Calendar](/x/react-date-pickers/accessibility/#date-calendar).
+
+### Date Range Calendar
+
+The [Date Range Calendar](/x/react-date-pickers/date-range-calendar/) implements a similar keyboard support as the day view of the [Date Calendar](/x/react-date-pickers/accessibility/#date-calendar) component, with a difference on the navigation among the previous and next months that must be achieved using the arrows in the calendar header.
+
+|                                                                                                                                          Keys | Description                                                   |
+| --------------------------------------------------------------------------------------------------------------------------------------------: | :------------------------------------------------------------ |
+| <kbd class="key">Arrow Up</kbd>, <kbd class="key">Arrow Down</kbd>,<br> <kbd class="key">Arrow Left</kbd>, <kbd class="key">Arrow Right</kbd> | Moves focus among the available values                        |
+|                                                                                                                <kbd class="key">Page Up</kbd> | Moves focus to the last day of the month                      |
+|                                                                                                              <kbd class="key">Page Down</kbd> | Moves focus to the first day of the month                     |
+|                                                                                                                   <kbd class="key">Home</kbd> | Moves focus to first day of the week within the current month |
+|                                                                                                                    <kbd class="key">End</kbd> | Moves focus to last day of the week within the current month  |
+
+### Date Range Picker
+
+When interacting with the keyboard, the [Date Range Picker](/x/react-date-pickers/date-range-picker/) keep the focus on the Field component, thereby offering the same keyboard navigation support as the [Date Range Field](/x/react-date-pickers/accessibility/#fields).
```

Comment:
```suggestion
When interacting with the keyboard, the [Date Range Picker](/x/react-date-pickers/date-range-picker/) keeps the focus on the Field component, thereby offering the same keyboard navigation support as the [Date Range Field](/x/react-date-pickers/accessibility/#fields).
```
P.S. WDYT about removing the `Date Range Field` or the `Field` link below on L84? 🤔 
They point to the same section and are too close to one another IMHO. 🙈

Reply:
I agree, and actually I think L82 and L84 are redundant